code/modules/1713/language_book.dm

```diff
@@ -56,7 +56,7 @@
 		langcopy.Remove(language1)
 		var/datum/language/language2 = WWinput(user, "What will your 2nd language to translate be?", "Writing Translations", "text", langcopy)
 		user.visible_message("<span class='notice'>[user] begins to write in the [src].</span>", "<span class='notice'>You begin writing translations in the [src]. This will take around 2 minutes, and you need to stand or sit still.</span>")
```

Comment:
Also, you didn't the strings sent to the person reading, you made the writing from 1200 (120 seconds) into 1800 (180 seconds)